### PR TITLE
Fix Broken Unit Tests Since Interop

### PR DESCRIPTION
**Pull request by ec2**

### CODE REVIEW

File: packages/lodestar/test/unit/api/rpc/api/validator/validator.test.ts [hunk 1]
```diff
@@ -18,25 +18,28 @@ import {generateEmptyAttestation} from "../../../../../utils/attestation";
...
     eth1Stub = sandbox.createStubInstance(EthersEth1Notifier);
     forkChoiceStub = sandbox.createStubInstance(StatefulDagLMDGHOST);
     getDutiesStub = sandbox.stub(validatorImpl, "getValidatorDuties");
     chainStub = sandbox.createStubInstance(BeaconChain);
     chainStub.forkChoice = forkChoiceStub;
+    chainStub.config = config;
     opStub = sandbox.createStubInstance(OpPool);
     opStub.attestations = sandbox.createStubInstance(AttestationOperations);
-    validatorApi = new ValidatorApi({}, {config, chain: chainStub, db: dbStub, opPool: opStub, eth1: eth1Stub});
+    validatorApi = new ValidatorApi({}, {config, chain: chainStub, db: dbStub, opPool: opStub, eth1: eth1Stub, logger: logger}); // need to add logger in here
```
Comment (GregTheGreek) [inline packages/lodestar/test/unit/api/rpc/api/validator/validator.test.ts]: Comment necessary?

File: packages/lodestar/test/unit/chain/stateTransition/util/misc.test.ts [hunk 2]
```diff
@@ -28,17 +28,17 @@ describe("getDomain", () => {
...
   it("epoch before fork epoch should result in domain === previous fork version * 2**32 + domain type", () => {
     const result = getDomain(config, state, 5, 13);
-    assert.equal(result.toString('hex'), '0000000500000005');
+    assert.equal(result.toString('hex'), '0500000005000000');
   });
 
   it("epoch before fork epoch should result in domain === previous fork version * 2**32 + domain type", () => {
     const result = getDomain(config, state, 5, 12);
-    assert.equal(result.toString('hex'), '0000000500000005');
+    assert.equal(result.toString('hex'), '0500000005000000');
```
Comment (GregTheGreek) [inline packages/lodestar/test/unit/chain/stateTransition/util/misc.test.ts:41]: What changed that caused these to flip?

File: packages/lodestar/test/unit/api/rpc/api/validator/validator.test.ts (hunk 1 above)
Comment (ec2) [reply packages/lodestar/test/unit/api/rpc/api/validator/validator.test.ts]: oops forgot to remove. thanks for that

File: packages/lodestar/test/unit/chain/stateTransition/util/misc.test.ts (hunk 2 above)
Comment (ec2) [reply packages/lodestar/test/unit/chain/stateTransition/util/misc.test.ts:41]: See: https://github.com/ChainSafe/lodestar/commit/4b1143fad7d7ce17c9f50b3f6948afb07bc02024#diff-375d2bcfea48c91c150b334eca6354aaL19

**Review by wemeetagain — APPROVED** [review]

**Review by GregTheGreek — APPROVED** [review]

**Pull request merged**